stdlib/json.gr

```diff
@@ -1068,11 +898,7 @@ let combineSurrogatePairToCodePoint =
   ((highSurrogate - 0xD800) << 10) + (lowSurrogate - 0xDC00) + 0x10000
 }
 
-let makePrettyPrintJsonWriter =
-  (
-    format: FormattingSettings,
-    buffer: Buffer.Buffer,
-  ) => {
+let makeJsonWriter = (format: FormattingSettings, buffer: Buffer.Buffer) => {
   let printNewLine = match (format.lineEnding) {
     NoLineEnding => () => void,
```

Comment:
These should not be functions anymore. You should put them inside Options so you know if you can skip calling the functions at all when they are noops.

Reply:
Done